kie-server-parent/kie-server-wars/kie-server/src/main/webc-resources/README.md

```diff
@@ -27,18 +31,61 @@ This instruction describes all steps to install KIE Server on Tomcat 9 standalon
  </tomcat-users>
 
  5. Configure data source for data base access by jBPM extension of KIE Server
-    Edit TOMCAT_HOME/conf/context.xml and add following within Context tags of the file
-
-       <Resource name="sharedDataSource"
-       		  auth="Container"
-       		  type="org.h2.jdbcx.JdbcDataSource"
-       		  user="sa"
-              password="sa"
-              url="jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;MVCC=TRUE"
-              description="H2 Data Source"
-              loginTimeout="0"
-              testOnBorrow="false"
-              factory="org.h2.jdbcx.JdbcDataSourceFactory"/>
+           
+    In order to configure a pooling XA data source, configure an XA data source without pooling capabilities first. 
+    This XA data source, named “xads” in the example below, is used for creating new connections to the target database.
+    
+    As a next step, configure a pooling data source, that relies on XA data source for creating new connections. 
+    In the example, this data source is named “poolingXaDs”.
+    
+    Edit TOMCAT_HOME/conf/context.xml and add following within Context tags of the file:
+    
+    <Resource 
```

Comment:
could you please enclose these in `source` block as it seems not to be rendered...

Reply:
done